posthog/api/early_access_feature.py

```diff
@@ -82,6 +83,22 @@ def create(self, validated_data):
 
         feature_flag_id = validated_data.get("feature_flag_id", None)
 
+        default_condition = [
+            {"properties": [], "rollout_percentage": 100, "variant": None},
+        ]
+        super_conditions = lambda feature_flag_key: [
+            {
+                "properties": [
+                    {
+                        "key": f"$feature_enrollment/{feature_flag_key}",
+                        "type": "person",
+                        "operator": "is_set",
```

Comment:
this should be `true` with operator exact?

Reply:
As otherwise the toggle on the opt in wouldn't really work. They wouldn't be able to opt out.

Reply:
I guess this becomes a misnomer, i'm not using `is_set` as an operator. The flag_matching logic just needs to know what key to look at